rocAL/rocAL/source/augmentations/geometry_augmentations/node_crop.cpp

```diff
@@ -27,42 +27,43 @@ THE SOFTWARE.
 #include "exception.h"
 
 CropNode::CropNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
-        Node(inputs, outputs),
-        _dest_width(_outputs[0]->info().max_shape()[0]),
-        _dest_height(_outputs[0]->info().max_shape()[1])
-{
+        Node(inputs, outputs) {
     _crop_param = std::make_shared<RocalCropParam>(_batch_size);
 }
 
-void CropNode::create_node()
-{
+void CropNode::create_node() {
     if(_node)
         return;
 
-    if(_dest_width == 0 || _dest_height == 0)
```

Comment:
Are we not validating with this check anymore?

Reply:
Not required / used